library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -576,7 +578,10 @@ data class Conversations(
     fun streamAllGroupMessages(): Flow<DecodedMessage> = callbackFlow {
         val messageCallback = object : FfiMessageCallback {
             override fun onMessage(message: FfiMessage) {
-                trySend(MessageV3(client, message).decode())
+                val decodedMessage = MessageV3(client, message).decode()
+                if (!(decodedMessage.encodedContent.type == ContentTypeGroupMembershipChange && decodedMessage.kind != MessageKind.MEMBERSHIP_CHANGE)) {
+                    trySend(decodedMessage)
+                }
```

Comment:
Would it make sense to move the filtering logic into `decode()`? Then it automatically applies to all call-sites, and can't be screwed up in future call-sites either

Reply:
Good call!